projects/components/CHANGELOG.MD

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+1. Add actionsUpdate emitter to ActionMenuComponent
```

Comment:
The rest of the file is using dashes, let's use that. There's no real order here so the numbers just add complexity if you re-order things.

Reply:
Please use  backticks around names of symbols such as `actionsUpdate`

Reply:
done